### DISCUSSION TITLE
how to find out snapshot size

### DISCUSSION THREAD

**Discussion by rusticek** [moosefs/moosefs#552] [General]
Hi,
i am using snapshot feature on MFS 3.0.116 and i am making snapshot every day (for 7 days) for several directories. I noticed dramatical data increase last weeks, but i can not find out, which files/dirs are growing/changing so much.

Is there a way to somehow compare changed chunks between live file and snapshoted file?

**Comment by chogata** [comment]
`mfsfileinfo` command shows chunk numbers. You could write a script comparing files in your live and snapshotted directory to see which files have chunks with different numbers.
